src/index.ts

```diff
@@ -6,9 +6,9 @@ import chalk from 'chalk';
 import ora from 'ora';
 import hasha from 'hasha';
 import yargs from 'yargs-parser';
-import resolveFrom from 'resolve-from';
 import babelPresetEnv from '@babel/preset-env';
 import isNodeBuiltin from 'is-builtin-module';
+import validatePackageName from 'validate-npm-package-name';
```

Comment:
@dangodev you are prescient. you merged this dependency into master 4 minutes before I needed it :)